workspace/src/contract.rs

```diff
@@ -387,6 +390,55 @@ impl<U: UserFunctions> EvmAccount<U> {
         CallStorageWithdraw(self.near_call(&Call::StorageWithdraw).args_json(args))
     }
 
+    pub fn factory_update(&self, bytes: Vec<u8>) -> CallFactoryUpdate<'_> {
+        CallFactoryUpdate(
+            self.near_call(&OwnerCall::FactoryUpdate)
+                .args_borsh(Raw(bytes)),
+        )
+    }
+
+    pub fn factory_set_wnear_address(
+        &self,
+        address: impl Into<Address>,
+    ) -> CallFactorySetWNearAddress<'_> {
+        let bytes = address.into().0;
+        CallFactorySetWNearAddress(
+            self.near_call(&OwnerCall::FactorySetWNEARAddress)
+                .args_borsh(Raw20(bytes)),
+        )
+    }
+
+    pub fn stage_upgrade(&self, bytes: Vec<u8>) -> CallStageUpgrade<'_> {
+        CallStageUpgrade(
+            self.near_call(&OwnerCall::StageUpgrade)
+                .args_borsh(Raw(bytes)),
```

Comment:
Don't we replace just with 
```suggestion
                .args(bytes),
```
?